Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -113,7 +113,10 @@ def validater(username, access_token):
 
         def prompt():
             result = "GitHub's API\nProvide {} username and access token to create and update pull requests".format(hostname)
-            if webkitcorepy.Terminal.open_url('{}?scopes=repo,workflow&description={}%20Local%20Automation'.format(token_url, self.name)):
+            if webkitcorepy.Terminal.open_url(
+                '{}?scopes=repo,workflow&description={}%20Local%20Automation'.format(token_url, self.name),
+                prompt='ENTER to open the GitHub token generation webpage: ',
```

Comment:
I feel like there should be a "please" on one of these lines :). Something like "Please press Enter to open GitHub token generation webpage. The options will be preconfigured, so you can just scroll to the bottom and submit the form.

Also, is it "webpage" or "web page"? I know we say "website", but not sure about page.

Reply:
I've seen both "web page" and "webpage", it looks like the WebKit project prefers "web page", but it doesn't appear to be a strong preference. Will change.

Reply:
Looking at https://www.apple.com/safari/, seems like "web page" indeed.